File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--protect-parts-of-documents-by-using-content-controls_3.cs
        private Microsoft.Office.Tools.Word.GroupContentControl groupControl1;

        private void ProtectFirstParagraph()
        {
            this.Paragraphs[1].Range.InsertParagraphBefore();
            Word.Range range1 = this.Paragraphs[1].Range;

            range1.Text = "You cannot edit or change the formatting of text " +
                "in this sentence, because this sentence is in a GroupContentControl.";
            range1.Select();
            groupControl1 = this.Controls.AddGroupContentControl("groupControl1");
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--protect-parts-of-documents-by-using-content-controls_3.vb
    Dim groupControl1 As Microsoft.Office.Tools.Word.GroupContentControl

    Private Sub ProtectFirstParagraph()
        Me.Paragraphs(1).Range.InsertParagraphBefore()
        Dim range1 As Word.Range = Me.Paragraphs(1).Range
        range1.Text = "You cannot edit or change the formatting of text " & _
                "in this paragraph, because this paragraph is in a GroupContentControl."
        range1.Select()

        groupControl1 = Me.Controls.AddGroupContentControl("groupControl1")
    End Sub